app/controllers/find_local_council_controller.rb

```diff
@@ -54,15 +57,37 @@ def locations_api_response
     @locations_api_response ||= fetch_location(postcode)
   end
 
+  def single_authority_from_location_api?
+    locations_api_response.local_custodian_codes == 1
+  end
+
+  def address_list
+    @address_list ||= build_addresses(postcode)
```

Comment:
Do we need to pass `postcode` through here? it looks like it's only used to `fetch_addresses`, and `build_addresses` is able to access postcode` directly anyway?